modules/apps/object/object-test/src/testFunctional/macros/ObjectSalesforce.macro

```diff
@@ -0,0 +1,37 @@
+definition {
+
+	macro configureSalesforce {
+		var salesforceLoginURL = PropsUtil.get("object.storage.salesforce.login.url");
+		var salesforceConsumerKey = PropsUtil.get("object.storage.salesforce.consumer.key");
+		var salesforceConsumerSecret = PropsUtil.get("object.storage.salesforce.consumer.secret");
+		var salesforceUsername = PropsUtil.get("object.storage.salesforce.username");
+		var salesforcePassword = PropsUtil.get("object.storage.salesforce.password");
```

Comment:
You probably need to run SF here, I think it's going to sort the vars alphabetically.